tool/tctl/common/tctl.go

```diff
@@ -26,6 +26,7 @@ import (
 	"strconv"
 
 	"github.com/gravitational/teleport"
+	apiclient "github.com/gravitational/teleport/api/client"
```

Comment:
```go
"github.com/gravitational/teleport/api/client"
```

Reply:
"github.com/gravitational/teleport/lib/client" is also imported